Fetching events 659 of 816 (389027491674555)
Traceback (most recent call last):
  File "scripts/get_times.py", line 99, in <module>
    main()
  File "scripts/get_times.py", line 95, in main
    get_all_times(args.input_file, args.id, args.headless)
  File "scripts/get_times.py", line 58, in get_all_times
    result = get_event_time(driver, event_id)
  File "scripts/get_times.py", line 20, in get_event_time
    driver.get("https://www.facebook.com/events/" + eid)
  File "/home/brugger/events.cbrugger/env/lib/python3.5/site-packages/selenium/webdriver/remote/webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "/home/brugger/events.cbrugger/env/lib/python3.5/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/home/brugger/events.cbrugger/env/lib/python3.5/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout
  (Session info: headless chrome=76.0.3809.100)
